fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1884,6 +1884,8 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 	Future<Void> openFuture;
 	std::unique_ptr<rocksdb::WriteBatch> writeBatch;
 	std::set<Key> keysSet;
+	// maximum number of single key deletes in a commit, if ROCKSDB_SINGLEKEY_DELETES_ON_CLEARRANGE is enabled.
+	int maxDeletes;
```

Comment:
It's probably cleaner that `maxDeletes` is part of `WriteBatch`, along with `keysSet`. This can be in another PR, though.

Reply:
Yes. That's what I meant.